gaia-header.js

```diff
@@ -1,9 +1,10 @@
+/* jshint node:true */
+/* globals define */
```

Comment:
IMO Gaia should not be linting outside dependencies :) If the project is not Mozilla owned then the coding convention will likely be different and linting will fail. You wouldn't lint your `node_modules` folder in a Node project.